src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -145,10 +145,14 @@ class ImageBaseViewer extends BaseViewer {
      * @return {boolean} value of whether discoverability is enabled for given type
      */
     isDiscoverabilityEnabled() {
-        const { enableAnnotationsImageDiscoverability, experiences = {} } = this.options;
-        const canShow = Object.values(experiences).some(experience => experience.canShow);
```

Comment:
This change was needed because:
- `experiences` structure changed
- We have another experience now, and I don't think this new experience's `canShow` should affect this method, while it'd do with the old implementation. So I refactored the method to depend only on `tooltipFlowAnnotationsExperience.canShow` (but not on `persistentOnboardingBoxEditAnnotationsExperience.canShow`)

Reply:
@megansmith-box, can you comment on the existing logic and if/how the new experience should integrate with it?

Reply:
@robertjk - This method was updated because for Images in Preview, we noticed that the Region is default Selected. We want to set the default selected to None, if this tooltip experience can show. We also want the tooltip to close when you click on the mode button. Is the new tooltip placed on the control bar? If so, I believe we do want to include it in this method, so we can leave it how it was. (If we rename dependencies back to experiences).

Reply:
@megansmith-box Thanks for that explanation.

> Is the new tooltip placed on the control bar? If so, I believe we do want to include it in this method, so we can leave it how it was.

Yes, the new tooltip displays on the same element as the previous one. So I reverted back to the old implementation, so that it affects the result of this method as well. Am resolving this.